src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -284,6 +284,14 @@ public function get_result_messages( $queue ) {
 				);
 			}
 
+			$images_scheduled = $queue->activity->get( 'images', 'scheduled' );
+			if ( ! empty( $images_scheduled ) ) {
+				$messages['success'][] = sprintf( // add image import count
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a2430c8c7a55dfe669c56e4deedd7e92</sub>